ingest/src/main/java/com/connexta/ingest/common/StreamingRestTemplate.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.common;
+
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+public class StreamingRestTemplate {
```

Comment:
❓ Should the `RestTemplate` `Bean` in `ApplicationConfiguration` be removed? Should we use the same `StreamingRestTemplate` for sending transform requests as well?